scene/gui/check_box.cpp

```diff
@@ -132,9 +132,9 @@ CheckBox::CheckBox(const String &p_text) :
 	set_text_alignment(HORIZONTAL_ALIGNMENT_LEFT);
 
 	if (is_layout_rtl()) {
-		_set_internal_margin(SIDE_RIGHT, get_icon_size().width);
-	} else {
 		_set_internal_margin(SIDE_LEFT, get_icon_size().width);
+	} else {
+		_set_internal_margin(SIDE_RIGHT, get_icon_size().width);
```

Comment:
This is wrong (in most cases `NOTIFICATION_THEME_CHANGED` will reset it back to correct value):
<img width="345" alt="Screenshot 2022-03-04 at 10 37 27" src="https://user-images.githubusercontent.com/7645683/156728928-0a52ea1d-bd4c-4261-904a-1bdafd075fc4.png">

Reply:
`CheckButton` have check box image on the other side of text, so margins should be in reverse.